examples/taskworkflow.go

```diff
@@ -56,30 +59,32 @@ func main() {
 
 func taskWorkflow(containerIP string, gateway string, netMask string) (err error) {
 	log.Println("Creating containerd client")
-	client, err := containerd.New("/run/containerd/containerd.sock")
+	client, err := containerd.New(containerdAddress)
 	if err != nil {
 		return errors.Wrapf(err, "creating client")
 
 	}
 	defer client.Close()
 	log.Println("Created containerd client")
 
-	ctx := namespaces.WithNamespace(context.Background(), "firecracker-containerd-example")
+	ctx := namespaces.WithNamespace(context.Background(), namespaceName)
 	image, err := client.Pull(ctx, "docker.io/library/nginx:latest",
 		containerd.WithPullUnpack,
 		containerd.WithPullSnapshotter("firecracker-naive"),
 	)
 	if err != nil {
 		return errors.Wrapf(err, "creating container")
+	}
 
+	pluginClient, err := ttrpcutil.NewClient(containerdAddress + ".ttrpc")
```

Comment:
nit: could add a `const containerdTTRPCAddress = containerdAddress + ".ttrpc"` at the top